backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -839,7 +841,12 @@ public FutureExperimentRunDAO(
                           return jdbi.withHandle(
                                   handle -> {
                                     var sql =
-                                        "select id, date_created, date_updated, experiment_id, name, project_id, description, start_time, end_time, owner, environment, code_version, job_id, version_number from experiment_run";
+                                        "select experiment_run.id, experiment_run.date_created, experiment_run.date_updated, experiment_run.experiment_id, experiment_run.name, experiment_run.project_id, experiment_run.description, experiment_run.start_time, experiment_run.end_time, experiment_run.owner, experiment_run.environment, experiment_run.code_version, experiment_run.job_id, experiment_run.version_number from experiment_run";
+
+                                    sql +=
+                                        " left join project p ON p.id = experiment_run.project_id ";
+                                    sql +=
+                                        " left join experiment e ON e.id = experiment_run.experiment_id ";
```

Comment:
I don't believe we need left joins, just inner joins, right?

Reply:
We do not want any data from right table then why inner join is required? we need just left table data with right table condition that's why i used left join. is it not current?

Reply:
Left joins allow the match _not_ to exist on the right table. In which case the entry on the left table will still be present in the result. So left joins _do not_ reduce the number of rows on the result, if I remember correctly. If we don't use the data on the right table, then left joins are effectively not doing anything besides wasting time.

If you just want to filter things out, use inner joins.

Reply:
Fix this with inner join